app/questionnaire/router.py

```diff
@@ -270,15 +271,31 @@ def _is_section_enabled(self, section):
         if "enabled" not in section:
             return True
 
-        for condition in section["enabled"]:
-            if evaluate_when_rules(
-                condition["when"],
+        enabled = section["enabled"]
+        if isinstance(enabled, Sequence):
```

Comment:
To be consistent with pathfinder, I think we should flip this logic. So check if it is a dict first then handle that, then handle list/tuple which is the old after.

Reply:
Logic flipped now, so first checking for dict and then `else`, thanks!